core/src/main/kotlin/motif/models/parsing/ObjectsClassParser.kt

```diff
@@ -125,6 +125,13 @@ class ObjectsClassParser : ParserUtil {
 
         val returnType: IrType = method.returnType
         val returnClass: IrClass = returnType.resolveClass() ?: throw TypeNotSpreadable(objectsClass, method, returnType)
+
+        returnClass.methods
+                .find { it.annotations.any { it.type.simpleName == "Nullable" } }
```

Comment:
I am using a `Nullable` string here to:

1. Avoid adding androidx as a compiler dependency.
2. Handling all different kinds of Nullable annotations at once (androidx, javax, jetbrains, etc.)

Reply:
Yep that's perfect was going to suggest this but you're way ahead of me :)